src/lib/mina_graphql/mina_graphql.ml

```diff
@@ -1213,11 +1291,36 @@ module Types = struct
                    |> Option.map ~f:(fun snapp_account ->
                           snapp_account.app_state |> Snapp_state.V.to_list
                           |> List.map ~f:Snapp_basic.F.to_string))
+             ; field "permissions" ~typ:account_permissions
+                 ~doc:"Permissions for updating certain fields of this account"
+                 ~args:Arg.[]
+                 ~resolve:(fun _ { account; _ } ->
+                   account.Account.Poly.permissions)
              ; field "tokenSymbol" ~typ:string
                  ~doc:"The token symbol associated with this account"
                  ~args:Arg.[]
                  ~resolve:(fun _ { account; _ } ->
                    account.Account.Poly.token_symbol)
+             ; field "verificationKey" ~typ:string
+                 ~doc:
+                   "Hash of the verification key associated with this account"
+                 ~args:Arg.[]
+                 ~resolve:(fun _ { account; _ } ->
+                   Option.value_map account.Account.Poly.snapp ~default:None
+                     ~f:(fun snapp_account ->
+                       Option.map snapp_account.verification_key ~f:(fun vk ->
+                           With_hash.hash vk
+                           |> Pickles.Backend.Tick.Field.to_string)))
```

Comment:
Shouldn't this use the encoding for verification keys, to allow for matching the input against a potential snapp?

Reply:
This is just the hash of the verification key which is good enough to check if vk was updated correctly. Do we want to show verification key as well?

Reply:
It seems odd that you set it with a transaction that contains the actual vk but then only get to confirm via the hash. We should probably have both.

Reply:
The transaction contains both the key and its hash https://github.com/MinaProtocol/mina/blob/c7c4c379e98c6078cd6122e1671099c66a966616/src/lib/mina_graphql/mina_graphql.ml#L2213

Reply:
Sure, but if you don't think to query it, and later want to check that the verification key was updated correctly, you have no way to get that hash.

Reply:
Fair enough. I'll add vk as well